fdbclient/S3BlobStore.actor.cpp

```diff
@@ -1129,13 +1129,19 @@ ACTOR Future<Reference<HTTP::IncomingResponse>> doRequest_impl(Reference<S3BlobS
 			}
 
 			Reference<HTTP::IncomingResponse> _r = wait(timeoutError(reqF, requestTimeout));
+			if (!_r) {
+				TraceEvent("S3BlobStoreDoRequestAfterDoRequestNoResponse")
+				    .detail("Verb", verb)
+				    .detail("Resource", resource)
+				    .detail("Timeout", requestTimeout);
+				continue;
```

Comment:
We should do wait(yield()) here to avoid busy loop.

Reply:
Added